### ISSUE TITLE
graphsage训练时loss为nan

### ISSUE THREAD

**Issue by chriskeke95** [alibaba/euler#263]
你好 分布式训练graphsage时 loss一直为nan，mrr不变，相同的数据训练line没问题，参数如下，调小lr到0.001也一样，请问是什么原因呢？
-aggregator='mean'
-batch_size=512
-dim=64
-feature_dim=3
-feature_idx=0
-learning_rate=0.01
-max_id=60000000
-mode='train'
-model='graphsage'
-num_epochs=1
-optimizer='sgd'
-share_negs=False
-unsupervised_loss='rank'

![image](https://user-images.githubusercontent.com/19768933/85491669-84a9c900-b606-11ea-80c2-9f0c72caeced.png)

**Comment by alinamimi** [comment]
-share_negs=False
-unsupervised_loss='rank'
这个应该不是我们的参数，是不是实现‘rank’ loss 有问题？

**Comment by chriskeke95** [comment]
> -share_negs=False
> -unsupervised_loss='rank'
> 这个应该不是我们的参数，是不是实现‘rank’ loss 有问题？

谢谢 修改unsupervised_loss = 'xent' 就可以了
